middle_end/clambda.mli

```diff
@@ -111,6 +111,7 @@ and ufunction = {
   dbg    : Debuginfo.t;
   env    : Backend_var.t option;
   mode   : Lambda.alloc_mode;
+  attrib : Lambda.check_attribute;
```

Comment:
It seems like this should be called `check` or `check_attrib`, or something?

Reply:
Renamed to `check`.